### ISSUE TITLE
Error when checking for local database

### ISSUE THREAD

**Issue by ferazambuja** [Mandrakia/Mandrasoft.PokemonGo#4]
Great work on the new interface!
The radar works, however, if I click to save locally I get this errors:
![screen shot](https://cloud.githubusercontent.com/assets/12782082/17163950/b1ef4862-5391-11e6-83ee-bfa8d2cd7a91.png)

**Comment by Mandrakia** [comment]
Yeah that functionality isn't implemented yet i'm still not sure which Db system to use :)

**Comment by ferazambuja** [comment]
Cool.

**Comment by Tropicocity** [comment]
I keep getting this at the moment 
![image](https://cloud.githubusercontent.com/assets/20605844/17193231/0e9d300c-544a-11e6-95fa-bda38be8cad5.png)
